bug-bounty-hof/client.yml

```diff
@@ -1,4 +1,74 @@
 names:
+- name: Sohom Datta
+  date: 2024-12-18
+- name: Narendra Bhati from Suma Soft Pvt. Ltd
+  date: 2024-12-18
+  twitter: "@imnarendrabhati"
+  url: https://twitter.com/@imnarendrabhati
+- name: Nan Wang
+  date: 2024-12-18
+  twitter: "@eternalsakura13"
+  url: https://twitter.com/@eternalsakura13
+- name: Dalmurino
+  date: 2024-12-11
+- name: Nils Bars
+  date: 2024-12-11
+- name: Irvan Kurniawan (sourc7)
+  date: 2024-12-04
+  url: https://www.linkedin.com/in/sourc7/
+- name: Paul Gerste
+  date: 2024-12-04
+- name: Jurrie Overgoor
+  date: 2024-11-27
+  url: https://www.linkedin.com/in/jurrieovergoor/
+- name: Dohyun Lee
+  date: 2024-11-20
+  twitter: "@l33d0hyun"
+  url: https://twitter.com/@l33d0hyun
+- name: Christian James
+  date: 2024-11-20
+- name: Lucas B. Williams
+  date: 2024-11-20
+- name: David & Johannes Henninger
```

Comment:
I think we want to list David and Johannes individually

Reply:
Done